docs/Debugging.md

```diff
@@ -8,24 +8,30 @@ next: testing
 ---
 
 ## Debugging React Native Apps
-To debug the javascript code of your react app do the following:
-
- 1. Run your application in the iOS simulator.
- 2. Press ```Command + D``` and a webpage should open up at [http://localhost:8081/debugger-ui](http://localhost:8081/debugger-ui). (Chrome only for now)
- 3. Enable [Pause On Caught Exceptions](http://stackoverflow.com/questions/2233339/javascript-is-there-a-way-to-get-chrome-to-break-on-all-errors/17324511#17324511) for a better debugging experience.
- 4. Press ```Command + Option + I``` to open the Chrome Developer tools, or open it via ```View``` -> ```Developer``` -> ```Developer Tools```.
- 5. You should now be able to debug as you normally would.
+To access the in-app developer menu, shake the iOS device or simulate a shake in the iOS Simulator by pressing `Control + Command + Z`.
```

Comment:
Let's use ⌘ instead of Command, and lowercase for the letter Z. I know it wasn't like this in the original, but it's more clear I think :smile: